src/components/package/create/package-create.js

```diff
@@ -123,7 +121,7 @@ class PackageCreate extends Component {
 }
 
 const validate = (values, props) => {
-  return Object.assign({}, validatePackageName(values, props), validateCoverageDates(values, props));
+  return Object.assign({}, validatePackageName(values), validateCoverageDates(values, props));
 };
 
 export default injectIntl(reduxForm({
```

Comment:
@cherewaty, I wanted to suggest for such compositions to use compose functions for example from redux.
I belive it would be much more readable, like:
compose(
injectIntl,
reduxForm({
...
})(PackageCreate)

what do you think?